sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3477,7 +3477,15 @@ object GpuOverrides extends Logging {
         TypeSig.ARRAY.nested(TypeSig.commonCudfTypesWithNested),
         TypeSig.ARRAY.nested(TypeSig.all)),
       (e, conf, p, r) => new GpuGetArrayStructFieldsMeta(e, conf, p, r)
-    )
+    ),
+    expr[RaiseError](
+      "throw exception",
```

Comment:
Nit: All other descriptions start with a capital letter and are a bit more descriptive, as seen in the generated configs.md docs.
```suggestion
      "Throws an exception",
```

Reply:
Thx, Done